webapp/channels/src/components/channel_bookmarks/channel_bookmarks_create_modal.tsx

```diff
@@ -104,51 +87,32 @@ function ChannelBookmarkCreateModal({
     }, [handleKeyDown]);
 
     // type === 'link'
```

Comment:
Ah, I see. I might've gone with making the modal with the name input into a separate component and then using that inside separate link and file modals rather than extracting the two behaviours into hooks, but I can see the reasoning for that. That might be me just not having used hooks enough yet 😅